src/frontend/src/pages/CalendarPage/DesignReviewCreateModal.tsx

```diff
@@ -127,16 +127,21 @@ export const DesignReviewCreateModal: React.FC<DesignReviewCreateModalProps> = (
 
   const memberOptions = users.map(userToAutocompleteOption);
 
+  const wbsDropdownOptionsSorted: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
```

Comment:
you don't need to variables defining the same thing. Have it as wbsDropdownOptions